src/generic/MPoly.jl

```diff
@@ -115,6 +115,41 @@ function derivative(f::MPoly{T}, x::MPoly{T}) where {T <: RingElement}
    return(derivative)
 end
 
+
+
+doc"""
+    change_base_ring(p::AbstractAlgebra.Generic.MPoly{T}, g) where {T <: RingElement}
+> Returns the polynomial obtained by applying g to the coefficients of p.
+"""
+function change_base_ring(p::AbstractAlgebra.Generic.MPoly{T}, g) where {T <: RingElement}
+   vars_parent = vars(p.parent)
+
+   n = p.parent.num_vars
+   exps = p.exps
+
+   size_exps = size(p.exps)
+   if (p.parent.ord != :lex)
```

Comment:
It's just code style, but in Julia you usually don't put brackets for if statements